hastexo/hastexo.py

```diff
@@ -318,24 +299,63 @@ def launch_stack_task_result(self, task_id):
 
     @XBlock.json_handler
     def get_user_stack_status(self, request_data, suffix=''):
-        configuration = self.get_configuration()
+        settings = get_xblock_settings()
+        course_id, student_id = self.get_block_ids()
 
         def _launch_stack(reset=False):
-            args = (
-                configuration,
-                self.stack_run,
-                self.stack_name,
-                self.get_stack_template(),
-                self.stack_user_name,
-                reset
-            )
+            stack_template = self.read_from_contentstore(
```

Comment:
What happens if this read fails? I see no exception handler here, which is generally fine — we do want to bail if the template can't be found. But would we get a stack trace somewhere?